container-toolkit/release-notes.md

```diff
@@ -10,6 +10,72 @@ This document describes the new features, improvements, fixed and known issues f
 
 ______________________________________________________________________
 
+## NVIDIA Container Toolkit 1.17.7
+
+This release of the NVIDIA Container Toolkit `v1.17.7` is a bugfix and minor feature release.
+
+### Known Issues
+
+There is a [known issue](https://github.com/NVIDIA/nvidia-container-toolkit/issues/1093) in this release that causes Enroot containers with Slurm to fail to load.
+This happens because container flags are discarded in cases where no `cuda-compat-mode` flag is specified in `nvidia-container-cli configure` command. 
```

Comment:
```suggestion
This happens because container flags are discarded when handling the default values of the `cuda-compat-mode` flag added to the `nvidia-container-cli configure` command in this release.
```